xml/System.Security.Permissions/SecurityPermissionFlag.xml

```diff
@@ -48,7 +48,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This enumeration has a <xref:System.FlagsAttribute> attribute that allows a bitwise combination of its member values.  
+ This enumeration has a <xref:System.FlagsAttribute> that allows a bitwise combination of its member values.  
```

Comment:
This information is already part of the summary in the rendered page. I think we can delete this.

```suggestion
```